flutter_local_notifications/example/android/app/src/debug/AndroidManifest.xml

```diff
@@ -1,5 +1,4 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.dexterous.flutter_local_notifications_example">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
Nice! Just checking -- is this intentional? I found this in [the `<manifest>` docs](https://developer.android.com/guide/topics/manifest/manifest-element): 

> In the Gradle-based build system, starting with AGP 7.3, don't set the package value in the source manifest file directly. For more information, see [Set the application ID](https://developer.android.com/studio/build/configure-app-module#set-application-id).

Reply:
I've run the Android Studio AGP Upgrade Assistant, any changes to .xml files came automatically, so good question.  In case take your edits :)

Reply:
@Levi-Lesches  from what I saw, what you sent aligns with the change that was done as the application ID has been specified in the `build.gradle`. If you create a new Flutter app using a relatively recent of the SDK then you should find it doesn't have a package value either (note: I don't recall whenever Flutter started behaving like this but I remember it's been like this for a while). With that in mind, I'm going to mark this as resolved